src/utils/file-processor.js

```diff
@@ -1,11 +1,9 @@
-import CryptoJS from "crypto-js";
 import Encryption from "utils/encryption";
 import _ from "lodash";
 
 import Iota from "../services/iota";
 
-const CHUNK_SIZE = Math.floor(0.7 * (2187 / 2)); // TODO: Optimize this.
-
+const CHUNK_SIZE = 1000;
```

Comment:
This could be up to 1060.  I left it at 1000, because I figured long term, being able to easily know how many chunks would be in a file based on the size might be more useful than fitting in 60 extra bytes per chunk.  But I'm open to changing this.